plugin.rb

```diff
@@ -120,17 +155,16 @@ def raw_info
   DiscourseEvent.on(:user_created) do |user|
     if SiteSetting.patreon_creator_enabled
       begin
-        nsfw_group = Group.find_by_name(SiteSetting.patreon_creator_nsfw_group)
-
-        user_info = Patreon.get("user_info") || {}
-        user_record = user_info[user.email] || {}
-        has_nsfw_campaign = user_record[:has_nsfw_campaign]
+        user_info_record = Patreon.get("user_info.#{user.email}") || {}
```

Comment:
Non-blocking: I could see `Patreon.get("user_info.#{user.email}")` moving to a helper like 

```ruby
def get_user_info_record(email)
    key = "user_info.#{email}"
    Patreon.get(key) || {}
end
```

Reply:
yeah I thought about it.